mysensors/handler.py

```diff
@@ -30,6 +30,11 @@ def handle_smartsleep(msg):
                     value_type,
                     new_value,
                 )
+                
+                
+def handle_wakeup(msg):
+    for child in msg.gateway.sensors[msg.node_id].children.values():
+        msg.gateway.sensors[msg.node_id].new_state = None
```

Comment:
We don't need to iterate anything. We should just clear the `new_state` dict. Please do this inside `handle_post_sleep_notification`. Use `dict.clear`.

https://docs.python.org/3/library/stdtypes.html#dict.clear